daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector-jaeger.md

```diff
@@ -0,0 +1,137 @@
+---
+type: docs
+title: "Using OpenTelemetry Collector to collect traces to send to Jaeger"
+linkTitle: "Using the OpenTelemetry for Jaeger"
+weight: 1200
+description: "How to push trace events to Jaeger distributed tracing platform, using the OpenTelemetry Collector."
+type: docs
+---
+
+Dapr supports writing traces using OpenTelemetry (OTLP) and Zipkin protocols, but for Jaeger, Zipkin support has been deprecated in favour of OpenTelemetry. Although Jaeger supports OTLP directly, the recommended approach for production is to use the OpenTelemetry Collector to collect traces from Dapr and send them to Jaeger, allowing your application to quickly offload data and take advantage of features like retries, batching, and encryption. For more information, read the Open Telemetry Collector [documentation](https://opentelemetry.io/docs/collector/#when-to-use-a-collector).
+
+## Configure Jaeger in self hosted mode
+
+### Local setup
+
+The simplest way to start Jaeger is to run the pre-built all-in-one Jaeger image published to DockerHub and expose the OLTP port:
+
+```bash
+docker run -d --name jaeger \
+  -p 4317:4317  \
+  -p 16686:16686 \
+  jaegertracing/all-in-one:1.49
+```
+
+Next, create the following YAML files locally:
+
+* **config.yaml**: Note that because you are using the Open Telemetry protocol
```

Comment:
```suggestion
> **Note:** Because you are using the Open Telemetry protocol to talk to Jaeger, you need to fill out the `otel` section of the tracing configuration and set the `endpointAddress` to the address of the Jaeger container.
```